projects/plugins/jetpack/tools/webpack.config.search.js

```diff
@@ -29,7 +46,7 @@ const baseWebpackConfig = getBaseWebpackConfig(
 			],
 		},
 		'output-chunk-filename': 'jp-search.chunk-[name]-[hash].js',
```

Comment:
Just a thought: does append `bundleVersionExtension` here make any sense?

Reply:
Do you mean the name doesn't make sense or add a version here doesn't? 
This solution does look a bit tricky, happen to use the proposal of George's.

Reply:
I was curious if it would be appropriate to set this to 'jp-search.chunk-[name]${ bundleVersionExtension }.js' or something similar. If we're investing time and maintenance burden into determining `bundleVersionExtension`, it might be good to deprecate hashes entirely if possible/appropriate.